### PR TITLE
Improve README and help provide an better project entry-point

### PR DESCRIPTION
**Pull request by vmayoral**

### CODE REVIEW

**Comment by ros-discourse** [comment]
This pull request has been mentioned on **ROS Discourse**. There might be relevant details there:

https://discourse.ros.org/t/interest-in-openembedded/33293/10

File: README.md [hunk 1]
```diff
@@ -1,37 +1,26 @@
...
 
-The original implementation has been retained in the
-[`legacy`](https://github.com/ros/meta-ros/tree/legacy) branch.
+- **Recipe Generation**: The project now uses recipes generated by [superflore](https://github.com/ros-infrastructure/superflore/). For insights into the transition and the associated development milestones, refer to this [wiki page](https://github.com/ros/meta-ros/wiki/Superflore-OE-Recipe-Generation-Scheme).
+
+- **Legacy Version**: If you're interested in the original implementation, it's preserved in the [`legacy`](https://github.com/ros/meta-ros/tree/legacy) branch.
+
+## Contributing
+
+Contributions to `meta-ros`` are always welcome! Whether it's submitting patches, reporting issues, or updating documentation, your efforts help improve the project.
```
Comment (LanderU) [inline README.md]:
```suggestion
Contributions to `meta-ros` are always welcome! Whether it's submitting patches, reporting issues, or updating documentation, your efforts help improve the project.
```

**Comment by vmayoral** [comment]
Thanks @LanderU!

**Comment by vmayoral** [comment]
No futher feedback received in here so merging for now. Happy to receive other improvements.

**Pull request merged**